common/config/rush/pnpm-lock.yaml

```diff
@@ -19192,6 +19192,7 @@ packages:
       '@types/mocha': 10.0.7
       '@types/node': 18.19.40
       '@types/sinon': 17.0.3
+      '@types/uuid': 8.3.4
```

Comment:
was this intended change?

Reply:
yes, this I have taken from GA3 for uuid generation (old pattern for generating the unique id)